test/src/e2e_vm_tests/test_programs/should_pass/language/u256/src/main.sw

```diff
@@ -26,6 +26,12 @@ fn not_operator() -> u256 {
     !(a - b)
 }
 
+// returns 8
+fn shift_operators() -> u256 {
```

Comment:
maybe add some corner cases here too? like shifting all bits out of a `u256`, making the least significant bit the most significant and vice versa

Reply:
Done,

Reply:
Ah, my bad, I should've used better wording. What I meant is something like `0x0...01 << 255` becomes `0x10....0`, and `0x10..0 >> 255` becomes `0x0...01`

Reply:
And for the "shifting out" all bits `0x1...11 << 256` becomes `0x0`, and `0x11..1 >> 256` becomes `0x0` as well

Reply:
Ah... Done.